TestIs2WavAudioPlayer/I2sWavPlayer.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Device.I2s;
+using System.Diagnostics;
+using System.IO;
+using nanoFramework.Hardware.Esp32;
+
+namespace AudioPlayer
+{
+    public class I2sWavPlayer : IDisposable
+    {
+        public enum Bus
+        {
+            One = 1,
+            Two = 2
+        }
+
+        private readonly I2sDevice _i2S;
+        private readonly FileStream _stream;
+
+        public I2sWavPlayer(Bus bus, string audioFile, int bckPin = 32, int dataPin = 33, int wsPin = 25)
```

Comment:
even if it's a sample, would be nice to add the /// intelisense comments with some explanation of each elements.